TesseractOCR/build_dependencies.sh

```diff
@@ -127,6 +129,28 @@ cd -
 rm -rf $GLOBAL_OUTDIR lib include
 
 #######################
+# Download dependencies
+#######################
+
+cd TesseractOCR
+git submodule init
+git submodule update
+
+#######################
+# Build libtiff and all of it's dependencies
+#######################
+
+cd libtiff-ios
```

Comment:
Instead of `cd`-ing into libtiff-ios and eventually `cd`-ing back out, how about just calling the scripts from the current directory? i.e. `./libtiff-ios/build-png.sh`, etc.